ddtrace/settings/config.py

```diff
@@ -552,6 +552,7 @@ def __init__(self):
 
         self._llmobs_enabled = asbool(os.getenv("DD_LLMOBS_ENABLED", False))
         self._llmobs_sample_rate = float(os.getenv("DD_LLMOBS_SAMPLE_RATE", 1.0))
+        self._llmobs_ml_app = os.getenv("DD_LLMOBS_APP_NAME")
```

Comment:
Maybe we should give users the ability to set ml-app in code as well for specific spans. For example if they have 3 different agent spans running within one environment, and they want each one to be a different ml app just so they can get clustering analysis at each of those levels

Reply:
Not saying that they should do this, but it would be interesting to see if they _do_ do it if we give them that option

Reply:
<img width="808" alt="Screenshot 2024-03-15 at 8 43 17 AM" src="https://github.com/DataDog/dd-trace-py/assets/35776586/b91a9851-90fa-45a5-b165-909bf2769c61">

Reply:
Oops didn't see that, sorry!